src/clj/web/chat.clj

```diff
@@ -30,11 +30,29 @@
         msg-cnt (mc/count db msg-collection {:username username :date {"$gt" start-date}})]
     (< msg-cnt max-cnt)))
 
-(defn- insert-msg [{{{:keys [username emailhash]} :user} :ring-req
-                    {:keys [:channel :msg]} :?data :as event}]
+(let [alpha (into #{} "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ")
+      rot13-map (->> (cycle alpha)
+                     (drop 26)
+                     (take 52)
+                     (zipmap alpha))]
+  (defn- rot13 [in]
+    (apply str (map #(get rot13-map % %) in))))
+
+(let [banned-words (->> (slurp "resources/public/chat/banned-words.txt")
```

Comment:
Do we want to store the file in a public place or can we put it elsewhere?

Reply:
We could put it anywhere, this was just my first thought. Placing it in /data was my other thought. The only issue is getting tests to pass if it's not being pulled from somewhere.